src/views/Game/Chat.tsx

```diff
@@ -235,7 +268,7 @@ export class GameChat extends React.PureComponent<GameChatProperties, any> {
                             ? _("Login to chat")
                             : !data.get('user').email_validated ? _("Chat will be enabled once your email address has been validated")
                                 : (this.state.chat_log === "malkovich"
-                                    ? pgettext("Malkovich logs are only visible after the game has ended", "Visible after the game")
+                                    ? pgettext("Malkovich logs are only visible to your opponent after the game has ended", "Visible after the game")
```

Comment:
I don't think we should churn this text for this as it's not displayed to the user ever - that's just a translator context string

Reply:
Ah that's fair.